test/CMakeLists.txt

```diff
@@ -91,40 +91,50 @@ foreach (test_src IN LISTS test_srcs)
   get_filename_component(test_name "${test_src}" NAME_WE)
   string(REGEX REPLACE "^test_" "" test_name "${test_name}")
 
-  # Some tests change behavior based on whether the compiler defs QUICK_TEST
-  # and/or QUICKER_TEST are defined. Detect these and build variants for each
+  # Some tests change behavior based on whether the compiler defs BENCHMARK
+  # and/or MINIMAL_TEST are defined. Detect these and build variants for each
   # configuration:
   cub_check_for_test_variants("${test_src}")
 
   foreach(cub_target IN LISTS CUB_TARGETS)
     if (NO_VARIANTS)
       # Only one version of this test.
       cub_add_test(test_target ${test_name} "${test_src}" ${cub_target})
+    elseif (NOT CUB_TEST_VARIANT STREQUAL "ALL")
+      string(TOLOWER CUB_TEST_VARIANT lowercase_cub_test_variant)
```

Comment:
This can be removed and rolled into the logic below if we switch to booleans options:
1. wrap the `thorough` logic in an `if (CUB_TEST_VARIANT_THOROUGH)`
2. update the `HAS_` checks to `if (CUB_TEST_VARIANT_XXX AND HAS_XXX_VARIANT)`

Reply:
Fixed in a slightly different way.